src/test/scala/io/abacus/pipeline/PipelineSpec.scala

```diff
@@ -1,37 +1,34 @@
 package io.abacus.pipeline
 
-import io.abacus.pipeline.Pipeline
-import org.scalatest.WordSpec
-import org.scalatest.ShouldMatchers
-import scala.collection.mutable.{Map =>  MMap, HashMap}
+import org.scalatest.{ShouldMatchers, WordSpec}
 
-class PipelineSpec extends WordSpec with ShouldMatchers {
-  // Simple WordCounter to test things
+import scala.collection.mutable
```

Comment:
I'm open to discussion. 
scala lang guide recommends this http://docs.scala-lang.org/overviews/collections/overview.html